apps/pdc-frontend/src/components/OpenFormsEmbed/OpenFormsEmbed.scss

```diff
@@ -66,3 +67,34 @@
 .utrecht-form-container--openforms {
   flex: 1 1;
 }
+
+.openforms-formio-content {
+  &--info {
+    --of-formio-content-info-border-inline-start-style: none;
```

Comment:
Kunnen deze design tokens niet op `.utrecht-theme`?

Reply:
Ik denk dat we hebben besloten dat alle Formio-gerelateerde stijlen in dit CSS-bestand moeten blijven, maar ik kan deze token naar `.utrecht-theme` verhuizen.

https://github.com/frameless/strapi/commit/93d0022547ee6a203747800b49ac788b7607eba2